src/thor/triplegbuilder.cc

```diff
@@ -30,9 +30,9 @@ using namespace valhalla::thor;
 
 namespace {
 
-void TrimShape(const float start,
+void TrimShape(float start,
                PointLL start_vertex,
-               const float end,
+               float end,
```

Comment:
woohoo! 👍

Reply:
This function also has potential failure when shape is empty. Maybe we need to defend against that. while(current != shape.end() - 1) this is going to blow off when shape is empty

Reply:
Fixed